### DISCUSSION TITLE
Is there a way to disable static page logging on build?

### DISCUSSION THREAD

**Discussion by mstudio** [vercel/next.js#36957] [Help]
We're using the[ Contentful Delivery API ](https://www.contentful.com/developers/docs/references/content-delivery-api/) to build our pages using ISR. Overall, this works great. My only issue at the moment is that the build process generates huge log files, as it dumps the entire JSON payloads from Contentful into the log file during the `Generating static pages` phase.

Is there any way to omit that from the build log?

Small sample of the JSON response that is logged...

```
video {
  metadata: { tags: [ [Object] ] },
  sys: {
    space: { sys: [Object] },
    id: 'YtmSlkjklklji0kjljU1UZkz',
    type: 'Entry',
    createdAt: '2022-05-07T23:38:03.395Z',
    updatedAt: '2022-05-16T19:06:00.627Z',
    environment: { sys: [Object] },
    revision: 16,
    contentType: { sys: [Object] },
    locale: 'ko-KR'
  },
  fields: {
    pageMetadata: { metadata: [Object], sys: [Object], fields: [Object] },
    l1Navigation: { metadata: [Object], sys: [Object], fields: [Object] },
    leadspace: { metadata: [Object], sys: [Object], fields: [Object] },
    videoId: '1_eoyqmbr4',
    duration: '10:14',
    thumbnail: { metadata: [Object], sys: [Object], fields: [Object] },
    econtentBanner: { metadata: [Object], sys: [Object], fields: [Object] },
    day: { metadata: [Object], sys: [Object], fields: [Object] },
    topics: [ [Object] ],
    title: 'foo"',
    slug: 'foo',
    description: { data: {}, content: [Array], nodeType: 'document' },
    resourcesSectionTitle: 'Resources',
    resourcesLinks: { data: {}, content: [Array], nodeType: 'document' },
    speakers: [ [Object] ],
    lever: [ [Object], [Object], [Object], [Object], [Object] ],
    relatedContent: [ [Object] ],
    languagesAvailable: [ [Object], [Object], [Object] ]
  }
}
```

**Comment by GabenGar** [comment]
Where does it dump exactly?

**Comment by icyJoseph** [comment]
How is this actually logging? I checked a few sites of mine, and well, though they get data from CMS, they don't seem to dump logs anywhere. All I see are the next.js build steps, and the summary of the build. 

I am going to take a wild guess and say that, your project logs the responses to the `stdout`, and somehow you pipe that into a log file.

**Reply by mstudio** [reply]
Thanks for checking your sites. I'm going to do a bit more digging and will report what I find. For production, we're using Travis -- but even without Travis, if I simply run `yarn build` on my local, the output in the terminal shows the logs as well -- all of the JSON from Contentful.

**Reply by Stikoun** [reply]
@mstudio Same issue for me, have you found some solution to disable logging?
